QueryGenie/AttributeProtocol.swift

```diff
@@ -35,7 +35,7 @@ public protocol NullableAttributeProtocol: AttributeProtocol {
 
 extension AttributeProtocol {
     
-    public final var ___expression: NSExpression {
+    public var ___expression: NSExpression {
```

Comment:
For all of the below lines, is there a good reason for removing `final`? These functions should really not be overridden.

Reply:
I am not sure. I will try to figure it out. You removed them in commit dfed2ae.

Reply:
`final` was removed because the complier throws an error: "only classes and class members may be marked with 'final'. "